src/utils/slack/creds.js

```diff
@@ -0,0 +1,18 @@
+const decrypt = (hash) => decodeURIComponent(window.atob(hash));
```

Comment:
I think creating a an extra folder, and a file for only this was not needed, a single `slack.js` file would have been enough. I don't expect to have much more code related to slack

Reply:
Hmm... yea... ok. I put `postErrorToSlack` together with this cred file because they're related to Slack-related functionality. However, it can be argued that most of the functionality in `postErrorToSlack` is not Slack specific. I'll remove it out.